koji/katello-mash-split.py

```diff
@@ -305,9 +305,9 @@ def main():
     
     branch_map = {
         'nightly': 'develop',
+        '3.15': '2.0',
         '3.14': '1.24',
-        '3.13': '1.23',
-        '3.12': '1.22',
```

Comment:
While the 3.12 jobs haven't been dropped, I'd keep this in since it doesn't hurt.